Bucketeer2/Sources/Internal/Model/Evaluation.swift

```diff
@@ -10,3 +10,31 @@ struct Evaluation: Hashable, Codable {
     let reason: Reason
     var variation_value: String
 }
+
+extension Evaluation {
+    func getVariationValue<T>(defaultValue: T, logger: Logger?) -> T {
+        let value = self.variation_value
+        let anyValue: Any?
+        switch defaultValue {
+        case is String:
+            anyValue = value
+        case is Int:
+            anyValue = Int(value)
+        case is Int64:
+            anyValue = Int64(value)
+        case is Float:
+            anyValue = Float(value)
+        case is Double:
+            anyValue = Double(value)
+        case is Bool:
+            anyValue = Bool(value)
+        default:
+            anyValue = value
```

Comment:
Is there a way to convert JSON objects in Swift to validate them?
By checking if it is a dictionary or not?

Reply:
Yes, it is possible.
I can also convert it to `[String: AnyHashable]` and check it.
Can I change [`BKTClient jsonVariation`](https://github.com/ca-dp/bucketeer-ios-sdk/pull/87/files/144e2a02474e5861ada9727fafb82a4683956c4c..4a221778df30bcdba879c7e6a7e2d8c5d69e5bf7#diff-ecc95802cf8595971f48fdbb7da52ef1181e037b9dd20132058a84345117f4e4R8) `Any` to `[String: AnyHashable])` if that is the case?

Reply:
As a trial, what do you think of the change? https://github.com/ca-dp/bucketeer-ios-sdk/pull/87/commits/68457f0ac275c5b57b521299716249a0a57c6553